.github/workflows/publish-keripy.yml

```diff
@@ -26,6 +26,18 @@ jobs:
         with:
           images: gleif/keri
 
+      - name: Set up Docker buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Cache Docker Layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: keri-${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            keri-${{ runner.os }}-buildx-
```

Comment:
Is something missing from the end of this line?

Reply:
No, it's just a match prefix. 

The reason it doesn't make sense to include the `${{ github.sha }}` value in match line (restore-keys) the line is because you only want to invalidate the Docker cache when the internals of the Docker Build command detect a change needed. 

The `${{ github.sha }}` value changes on every new commit which would mean an entirely new cache folder on every commit which defeats the purpose of caching. The value you can gain with caching only comes from the intermediary Docker image layers prior to the COPY command for copying source code in.

Typically this only happens when you make a change to the Dockerfile or when something copied in with a COPY command changes.